style.css

```diff
@@ -63,4 +72,238 @@
     list-style: none;
     padding: 0 2rem 0 1rem;
   }
+
+  .main-Grid {
+    display: grid;
+    grid-template-columns: 100%;
+    row-gap: 8rem;
+  }
+
+  .snapshoot {
+    display: flex;
+    width: 100%;
+    height: auto;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    padding: 1rem 1rem 3rem 1rem;
+    border-radius: 2rem;
+    margin: 0 1.5rem 0 1.5rem;
+  }
```

Comment:
- [ ] [REQUIRED] You are doing great sticking to the design of your chosen [template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). However, as you can see from below presented comparison, there are some things that really need improvement: 

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| ![Screenshot from 2022-02-13 02-20-19](https://user-images.githubusercontent.com/67608876/153728819-b297210d-1e02-4405-84bd-72d337af0f78.png) | ![Screenshot from 2022-02-13 02-19-18](https://user-images.githubusercontent.com/67608876/153728793-f0b67c6c-fa01-42d3-a12b-d42b6b779e5e.png)|

Specifically, you need to:

- [ ] Make sure your `<button/>` has an appropriate width. The easiest way of doing that actually can be done here in the card container `.works`. All you have to do is to add `align-items: flex-start;` and remove the `width: 30%;` from your `<button/>` element which has the class of  `.btn-Project`   
- [ ] Please make sure you have respective `line-height` for your `<p/>` with the class name `description-Text` element
you can get the value from the Figma

Reply:
completed